stable/external-dns/templates/deployment.yaml

```diff
@@ -1,10 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels: {{ include "external-dns.labels" . | indent 4 }}
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
   name: {{ template "external-dns.fullname" . }}
 spec:
+  selector:
+    matchLabels: {{ include "external-dns.labels" . | indent 6 }}
```

Comment:
indent is not needed here

Reply:
I am not sure what do you mean by this. Removing ident would cause yaml parse error.

with ident:
```yaml
spec:
  selector:
    matchLabels:
      app: external-dns
      heritage: Tiller
      release: release-name

```
without ident:
```yaml
spec:
  selector:
    matchLabels:
app: external-dns
heritage: Tiller
release: release-name
```

Am I missing something here?